### ISSUE TITLE
Prevent ingesting low quality pictures.

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/image-service#46]
From freshdesk ticket https://support.ehelp.edu.au/a/tickets/27235

> There are many pictures that are absolutely useless because they are completely out of focus or are much too low in resolution. (I think submitting photographers should be encouraged to provide photos that demonstrate the diagnostic features for that species). All photos should be vetted prior to inclusion. There are also some entries that are misplaced.

**Comment by djtfmartin** [comment]
I dont think we should prevent ingest of low resolution images, but i think we should lower the priority of them in displays such as species pages.
The API allows retrieval of image sizes so Im going to transfer this to BIE as a future enhancement of the species page gallery.
